.travis.yml

```diff
@@ -39,15 +39,12 @@ matrix:
   include:
     - rvm: jruby-1.7
       env: JRUBY_OPTS='--dev --1.8'
-    - rvm: 2.7.1
-      env: DIFF_LCS_VERSION="~> 1.3.0"
   allow_failures:
     - rvm: jruby-head
     - rvm: ruby-head
-    - rvm: ruby-3.0.0-preview1
     - rvm: rbx-3
   fast_finish: true
 branches:
```

Comment:
Wondering if this is correct at all. What is this for? I see both `pr` and `push` builds. Is this supposed to trigger a `push` build only for certain branches? I couldn't find this in the doc.
There's this, but it's still not clear how to exclusively run the `pr` build https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

Reply:
PR's always run, builds can also run on pushes to branches.